src/Tizen.NUI.Components/Controls/NumberPicker.cs

```diff
@@ -0,0 +1,531 @@
+        /// <param name="style">Creates NumberPicker by special style defined in UX.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NumberPicker(string style) : base(style)
+        {
+            Initialize();
+        }
+
+        /// <summary>
+        /// Creates a new instance of NumberPicker.
+        /// </summary>
+        /// <param name="numberPickerStyle">Creates NumberPicker by style customized by user.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public NumberPicker(NumberPickerStyle numberPickerStyle) : base(numberPickerStyle)
+        {
+            Initialize();
+        }
+
+        /// <summary>
+        /// Dispose NumberPicker and all children on it.
+        /// </summary>
+        /// <param name="type">Dispose type.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+
+            if (type == DisposeTypes.Explicit)
+            {
+                if (itemList != null)
+                {
+                    foreach (TextLabel textLabel in itemList)
+                    {
+                        if (pickerScroller) pickerScroller.Remove(textLabel);
+                        Utility.Dispose(textLabel);
+                    }
+
+                    itemList = null;
+                }
+
+                if (pickerScroller != null)
+                {
+                    Remove(pickerScroller);
+                    Utility.Dispose(pickerScroller);
+                    pickerScroller = null;
+                }
+
+                Remove(upLine);
+                Utility.Dispose(upLine);
+                Remove(downLine);
+                Utility.Dispose(downLine);
+            }
+
+            base.Dispose(type);
+        }
+
+        /// <summary>
+        /// An event emitted when NumberPicker value changed, user can subscribe or unsubscribe to this event handler.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<NumberPickerValueChangedEventArgs> NumberPickerValueChanged;
+
+        //TODO Fomatter here
+
+        /// <summary>
+        /// The values to be displayed instead of numbers.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ReadOnlyCollection<String> DisplayedValues
+        {
+            get
+            {
+                return displayedValues;
+            }
+            set
+            {
+                displayedValues = value;
+
+                needItemUpdate = true;
+                displayedValuesUpdate = true;
+
+                updateValueList();
+            }
+        }
+        
+        /// <summary>
+        /// The Current value of NumberPicker.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int PickerValue
```

Comment:
In `Progress`, it uses API names `MaxValue`, `MinValue` and `CurrentValue`.
How about changing `PickerValue` to `CurrentValue`.

Reply:
Agree withe that